### ISSUE TITLE
Support for exporting the images in dark mode (with black background)

### ISSUE THREAD

**Issue by reapedjuggler** [excalidraw/excalidraw#4689]
Currently, when a user tries to export the work it only supports exporting in light theme only instead if the user opts for exporting in dark mode (that is with black background) then it should be exported in the desired manner.

P.S: Great application team GG :smile: 
Some images are attached for more ref:

Here is the screenshot of the **actual image**
![excali1](https://user-images.githubusercontent.com/52465652/152098669-64f300bf-3abe-4183-9d40-1ee15fd52dd7.png)

Here is the screenshot of the **exported image**
![excali2](https://user-images.githubusercontent.com/52465652/152098675-8ed3357f-8742-4cad-8815-b00b546cebf5.png)

**Comment by h7y** [comment]
It is possible to export in dark mode. In the export modal, the toggle has to switched to dark mode.

![firefox_D96vnWuVDm](https://user-images.githubusercontent.com/22396000/152280396-681a585e-778d-467f-b209-88befbe7adc9.gif)

**Comment by reapedjuggler** [comment]
Oh totally missed it, btw can we somewhat increase the size of the logo as it seems small relatively?
Browser: Chrome

**Comment by ad1992** [comment]
Yep we definitely need to improve the visibility of the theme icon - this is part of the export redesign as well. And we should probably export in same theme which is set by user, IOW if its dark theme -> export in dark theme as well

**Comment by reapedjuggler** [comment]
Appreciate your quick response over this :), yeah it can be exported in the same mode by default and maybe there can be a manual setting too (as it's for now)

**Comment by dwelle** [comment]
>  And we should probably export in same theme which is set by user, IOW if its dark theme -> export in dark theme as well

This was actually discussed at length before and the team got divided on how this should work. I myself am for not aligning it to the UI theme, though I acknowledge the potential UX issues (the problem is, neither approach is free of UX issues).

**Comment by ad1992** [comment]
> > And we should probably export in same theme which is set by user, IOW if its dark theme -> export in dark theme as well
> 
> This was actually discussed at length before and the team got divided on how this should work. I myself am for not aligning it to the UI theme, though I acknowledge the potential UX issues (the problem is, neither approach is free of UX issues).

Whats the reason for not aligning with the UX theme ? From my understanding its more likely that user will want to export the data in same theme in which they are drawing. We can take a look at the data probably before taking the call.
